templates/admin/incidents.html.twig

```diff
@@ -0,0 +1,23 @@
+{% extends "deliveries.html.twig" %}
+
+{% block breadcrumb %}
+{{ parent() }}
+<li>{% trans %}adminDashboard.incidents.title{% endtrans %}</li>
+{% endblock %}
+
+{% block content %}
+  <div class="text-right mb-4">
+    <a href="" class="btn btn-success" onClick="alert('Only available for QACoops for now')">
```

Comment:
This `alert()` couples the app with our own internal release system. Also, this doesn't seem to do anything. 
Should be removed IMHO.